teacher.rb

```diff
@@ -0,0 +1,7 @@
+require_relative('./person')
+
+class Teacher < Person
+  def can_use_services?
+    true
+  end
+end
```

Comment:
- [ ] Your `Teacher` class should have a constructor that extends the parent's constructor by adding @specialization and a parameter for it, [Check out this lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars) to learn more about constructors, Your constructor will look something like this :point_down: 

```ruby
  def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
  end
```

Reply:
Thank you